rosdep/python.yaml

```diff
@@ -1599,6 +1599,13 @@ python-imaging:
     yakkety_python3: [python3-imaging]
     zesty: [python-imaging]
     zesty_python3: [python3-imaging]
+python-imaging-imagetk:
+  ubuntu:
+      artful: [python-pil.imagetk]
```

Comment:
This indentation is incorrect. And please use the generic `ubuntu: [python-pil.imagetk]` since all versions are the same.